<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TyponOptionsWidget</class>
 <widget class="QWidget" name="TyponOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MYUnitsSpinBox" name="typonWidthSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelHeight">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MYUnitsSpinBox" name="typonHeightSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelThickness">
         <property name="text">
          <string>Copper thickness:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="copperThickNessSpinBox">
         <property name="suffix">
          <string>Âµm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYUnitsSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">utils/myunitsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
